### PR TITLE
HomeWork1 task

### PR DESCRIPTION
**Pull request by norsulfazol**

### CODE REVIEW

File: Tasks/HomeWork1/homework1.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            if not new_value:
+                params[item] = params[item]['default_value']
+                break
+            if isinstance(params[item]['default_value'], Decimal):
+                if is_decimal(new_value):
+                    params[item] = Decimal(new_value)
+                    break
+                print('\rНеобходимо указать число!', end='')
+                sleep(1.5)
+                print('\r', end='')
```
Comment (MikitaTsiarentsyeu) [inline Tasks/HomeWork1/homework1.py:45]: Why so complicated? Basic print('Необходимо указать число!') would do the same, apart from the 1.5 second waiting which purpose is not quite clear for me.

**Comment by MikitaTsiarentsyeu** [comment]
I would like to see your tasks in a separated folder with your name on it

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
Please move your file to a specific folder with your name on it.

File: Tasks/HomeWork1/homework1.py (hunk 1 above)
Comment (norsulfazol) [reply Tasks/HomeWork1/homework1.py:45]: The user is warned about the need to enter the correct value. After a short delay, a carriage return to the beginning of the line occurs, that is, the warning line is overwritten. Finally, on the same line, you are prompted again to enter a value.

**Pull request closed without merging**